packages/e2e-tests/src/utils.ts

```diff
@@ -164,21 +163,26 @@ export class BuildSetup {
   }
 
   public getProcessData = () => {
-    let dataDirPath: string
-    let resourcesPath: string
+    let dataDirPath: string = ''
+    let resourcesPath: string = ''
     const backendBundlePath = path.normalize('backend-bundle/bundle.cjs')
-    const byPlatform = {
+    const byPlatform: { [key: string]: string } = {
       linux: `pgrep -af "${backendBundlePath}" | grep -v egrep | grep "${this.dataDir}"`,
       darwin: `ps -A | grep "${backendBundlePath}" | grep -v egrep | grep "${this.dataDir}"`,
-      win32: `powershell "Get-WmiObject Win32_process -Filter {commandline LIKE '%${backendBundlePath.replace(/\\/g, '\\\\')}%' and commandline LIKE '%${this.dataDir}%' and name = 'Quiet.exe'} | Format-Table CommandLine -HideTableHeaders -Wrap -Autosize"`
+      win32: `powershell "Get-WmiObject Win32_process -Filter {commandline LIKE '%${backendBundlePath.replace(
+        /\\/g,
+        '\\\\'
+      )}%' and commandline LIKE '%${
+        this.dataDir
+      }%' and name = 'Quiet.exe'} | Format-Table CommandLine -HideTableHeaders -Wrap -Autosize"`
     }
     const command = byPlatform[process.platform]
```

Comment:
You can probably use here `byPlatform[process.platform as SupportedPlatform]` instead of `const byPlatform: { [key: string]: string }` above